src/uvw/stream.hpp

```diff
@@ -155,8 +155,8 @@ class StreamHandle: public Handle<T, U> {
 
 public:
 #ifdef _WIN32
-    StreamHandle(ConstructorAccess ca, std::shared_ptr<Loop> ref)
-        : Handle{ca, std::move(ref)}
+    StreamHandle(typename Handle<T, U>::ConstructorAccess ca, std::shared_ptr<Loop> ref)
+        : Handle<T, U>{std::move(ca), std::move(ref)}
```

Comment:
What problem should this solves exactly? What error are you having?  
Set aside the fact that `ConstructorAccess` is a tag that doesn't require to be moved, it's already a custom part aimed to get compilation working on Windows and it just works on `appveyor` as expected.  
As far as I can see, this change isn't required.